python/glow/gwas/lin_reg.py

```diff
@@ -88,6 +93,9 @@ def linear_regression(genotype_df: DataFrame,
         - ``tvalue``: The T statistic
         - ``pvalue``: P value estimated from a two sided T-test
         - ``phenotype``: The phenotype name as determined by the column names of ``phenotype_df``
+        - ``n``: (verbose_output only) number of samples with non-null phenotype
+        - ``sum_x``: (verbose_output only) sum of genotype inputs, X
+        - ``y_transpose_x``: (verbose_output only) dot product of phenotype response and genotype input
```

Comment:
please mention the data structure of these new results. For example it seems y_transpose_x is a flattened array. Also need to explain precisely what happens to missing values in this output perhaps with an example.

Reply:
There are no complex structures here, but hopefully an additional blurb and type info is sufficient.